core/src/mindustry/world/blocks/campaign/LaunchPad.java

```diff
@@ -200,7 +200,7 @@ public void draw(){
 
             Draw.z(Layer.weather - 1);
 
-            TextureRegion region = Core.atlas.find("launchpod");
+            TextureRegion region = ((mindustry.world.blocks.campaign.LaunchPad)blockOn()).podRegion;
```

Comment:
```suggestion
            TextureRegion region = blockOn() instanceof LaunchPad p ? p.podRegion : Core.atlas.find("launchpod");
```

Reply:
You don't see it?
`public @Load(value = "@-pod", fallback = "launchpod") TextureRegion podRegion;`

Reply:
This good idea but - no

Reply:
What if the launch pod flies up and the pad gets deleted? Your code will crash.

Reply:
Hm... Okay